### ISSUE TITLE
confusion about the Recently Completed pane

### ISSUE THREAD

**Issue by gibsonjc** [CARLI/web-reports#76]
Recently completed pane: It took me a minute to realize that only clicking the Result text or icon would show the report output. More accurately, I was momentarily put off when clicking the query name didn’t change the screen to the results. I don’t really have a suggestion for this yet, in part because I found what I needed by clicking around. Maybe indenting the query name or giving it an icon or even just having “query:” in front of the query name.

**Comment by gibsonjc** [comment]
Try putting the paper/sheet icon in front of the report name in the Recently Completed box.

**Comment by patrickzurek** [comment]
I added the file icon to the front of the report template name.  I also changed the icon above it from an area chart icon to a table icon.  There were two reasons for this.  1) I felt a table icon better described what was being linked to and 2) The area chart was mostly a solid color and I didn't think it looked that great positioned above the sparsely colored file icon.  The new table icon does not have a mostly colored background.
